README.md

```diff
@@ -10,12 +10,115 @@ Installation
 ember install gavant-ember-app-version-update
 ```
 
+Add the settings below (shown with default values) to your app's config/environment.js to configure the addon. At the very least, you **must** set the `versionUpdate.version` value to a string of `"BUILD_VERSION"`. This is required, as it is a placeholder that is replaced during deployment with the application's actual current version number.
+
+```js
+let ENV = {
+
+    //other environment configs...
+
+    versionUpdate: {
+        //REQUIRED - it must defined with a value of exactly "BUILD_VERSION"
+        version: 'BUILD_VERSION',
+        //the response header sent by your API which contains the current app version value
+        header: 'x-web-version',
+        //the websocket event type that contains a new app version value
+        socketEventType: 'webVersionChange',
+        //A delay (in milliseconds) to wait before automatically refreshing the user's browser
+        //when a new app version is received
+        refreshDelay: 60000
+    }
+}
+```
 
 Usage
 ------------------------------------------------------------------------------
+By default, gavant-ember-app-version-update provides no user-facing UI to notify when a new version is received, other than automatically reloading the app after the auto-refresh timer expires.
+
+However, you will almost definitely want to show the user some sort of notification that a version update is about to happen (an potentially allow them to cancel the automatic refresh). You can do this with the `VersionUpdate` service and its computed properties and events that it fires.
+
+For example, to show a toast notification with [ember-cli-notifications](https://github.com/stonecircle/ember-cli-notifications):
+
+```js
+//application/route.js
+export default Route.extend({
+    versionUpdate: service(),
+    fastboot: service(),
+    notifications: service('notification-messages')
+
+    afterModel() {
+        this._super(...arguments);
+        get(this, 'versionUpdate').on('versionChanged', this, () => {
+            if(!get(this, 'fastboot.isFastBoot')) {
```

Comment:
@billdami Im thinking it might be smarter for us to handle fastboot in the addon. I can't imagine you would ever want to send out a notification when rendered in fastboot